src/components/item/ItemPreviewCard.vue

```diff
@@ -112,9 +110,12 @@
       imageUrl() {
         const size = 'w400';
 
-        return this.value.edmPreview ?
-          `${this.value.edmPreview[0]}&size=${size}` :
-          genericThumbnail(this.value.id, { type: this.value.type, size });
+        if (this.value.edmPreview) {
+          const url = new URL(this.value.edmPreview[0]);
+          return this.$apis.thumbnail.url(url.searchParams.get('uri'), url.searchParams.entries());
```

Comment:
I don't see the search parameters being set in the returned url. I think because `.entries()` returns an iterator, not an object as the `url` function expects. https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/entries
And shouldn't we add the size parameter here too?

Reply:
I have refactored further to make a new `edmPreview` function on the thumbnail plugin, which sets all of type, size and uri, and extended the unit tests to cover.